tests/iostability/test_iostability_happy_path.py

```diff
@@ -87,3 +92,46 @@ def test_bucket_object_crud_s3bench(self):
         self.test_completed = True
         self.log.info(
             "ENDED: Test for Bucket and  Object CRUD operations in loop using S3bench for 30 days")
+
+    @pytest.mark.lc
+    @pytest.mark.io_stability
+    @pytest.mark.tags("TEST-40041")
+    def test_disk_near_full_s3bench(self):
+        """Perform disk storage near full once and read in loop for 30 days."""
+        self.log.info("STARTED: Perform disk storage near full once and read in loop for 30 days.")
+        self.log.info("Step 1: calculating byte count for required percentage")
+        resp = self.dfr.get_user_data_space_in_bytes(master_obj=self.master_node_list[0],
+                                                     memory_percent=95)
+        if resp[1] != 0:
+            self.log.info("Need to add %s for required percentage", resp[1])
+            self.log.info("Step 2: performing writes till we reach required percentage")
+            s3userinfo = dict()
+            s3userinfo['accesskey'] = ACCESS_KEY
+            s3userinfo['secretkey'] = SECRET_KEY
+            bucket_prefix = "testbkt"
+            ret = DiskFailureRecoveryLib.perform_near_full_sys_writes(s3userinfo=s3userinfo,
+                                                                      user_data_writes=int(resp[1]),
+                                                                      bucket_prefix=bucket_prefix,
+                                                                      client=20)
+            assert_utils.assert_true(ret[0], ret[1])
+            self.log.debug("write operation data: %s", ret)
```

Comment:
do we have an return from the above method on how much time it took just for info logger?

Reply:
Yes, we can add in lib level, will take it as a improvement in next PR.